Assignment1.Tests/RegExprTests.cs

```diff
@@ -2,4 +2,108 @@ namespace Assignment1.Tests;
 
 public class RegExprTests
 {
+    [Fact]
+    public void SplitLine_Given_Stream_Of_Lines_Returns_Stream_Of_Words()
+    {
+        // Arrange
+        var lines = new[] { "Hej", "med", "dig", "jeg hedder Bob!", "Jeg er 55 år." };
+
+        var words = new[] { "Hej", "med", "dig", "jeg", "hedder", "Bob", "Jeg", "er", "55", "år" };
+
+        // Act
+        var actual = RegExpr.SplitLine(lines);
+
+        // Assert
+        Assert.Equal(words, actual);
+    }
+
+    [Fact]
+    public void SplitLine_Given_Empty_Stream_Returns_Empty_Stream()
+    {
+        // Arrange
+        var lines = Array.Empty<string>();
+
+        var words = Array.Empty<string>();
+
+        // Act
+        var actual = RegExpr.SplitLine(lines);
+
+        // Assert
+        Assert.Equal(words, actual);
+    }
+
+    [Fact]
+    public void Resolution_Given_String_Of_Resolutions_Returns_Stream_Of_Resolutions()
+    {
+        // Arrange
+        var input = new[] { "1920x1080, 1024x768, 800x600, 640x480, 320x200, 320x240, 800x600, 1280x960"} ;
```

Comment:
For at gøre det til en bedre test case, så lav flere entries i det array, i stedet for at input[0] har det hele